clients/drcachesim/tracer/syscall_pt_trace.cpp

```diff
@@ -125,9 +130,9 @@ syscall_pt_trace_t::start_syscall_pt_trace(IN int sysnum)
         return false;
     }
 
-    /* All syscalls within a single thread share the same pttracer configuration, and
-     * thus, the same pt_metadata. Metadata is dumped at the beginning of the output
-     * file and occurs only once.
+    /* All syscalls within a single thread share the same pttracer configuration,
+     * and thus, the same pt_metadata. Metadata is dumped at the beginning of the
+     * output file and occurs only once.
```

Comment:
Not seeing any change here? Accidental reformat?

Reply:
Not sure. I assumed it was clang-format's doing. Reverted.